parity/cli/mod.rs

```diff
@@ -463,6 +463,10 @@ usage! {
 
 
 		["API and Console Options – HTTP JSON-RPC"]
+			FLAG flag_allow_missing_blocks: (bool) = false, or |c: &Config| c.rpc.as_ref()?.allow_missing_blocks.clone(),
```

Comment:
```suggestion
			FLAG flag_jsonrpc_allow_missing_blocks: (bool) = false, or |c: &Config| c.rpc.as_ref()?.allow_missing_blocks.clone(),
```

I think we should keep args to be `--jsonrpc-allow-missing-block`, but set configs to be only `allow_missing_blocks`, because the later can be identified inside the `rpc` category already.